src/tests/e2e/address-mapper-tests.ts

```diff
@@ -10,12 +10,14 @@ import {
 } from '../../index'
 import { createTestHttpClient } from '../helpers'
 import { Web3Signer } from '../../solidity-helpers'
+import { ethers } from 'ethers';
 
 const Web3 = require('web3')
 
 // TODO: Need a factory to create connection properly likes plasma-cash test
 function getWeb3Connection() {
-  return new Web3('http://127.0.0.1:8545')
+  const provider = new ethers.providers.JsonRpcProvider('http://localhost:8545')
+  return provider.getSigner()
```

Comment:
Can you renamed this method to something more appropriate now that it doesn't actually return a Web3 instance?

Reply:
Yes